src/components/TooltipTrigger.js

```diff
@@ -150,7 +171,7 @@ export default class TooltipTrigger extends Component {
           this.setState({ open: false })
         }
       }),
-      open && h(Tooltip, { target: this.id, ...props }, [content])
+      open && h(Tooltip, { target: this.id, type, ...props }, [content])
```

Comment:
FYI: Since we're using the rest/spread operator here to pass down additional props, the `type` prop would already get passed through to the `Tooltip` by default. But being explicit is also fine, and you could argue it's easier to understand.